Exercises-4-OOP/src/ExercisesOnClasses/RectangleTest.java

```diff
@@ -0,0 +1,27 @@
+package ExercisesOnClasses;
+
+public class RectangleTest {
+
+	public static void main(String[] args) {
```

Comment:
The output should be:
```
Rectangle[length=1.2,width=3.4]
Rectangle[length=1.0,width=1.0]
Rectangle[length=5.6,width=7.8]
length is: 5.6
width is: 7.8
area is: 43.68
perimeter is: 26.80
```

But in this case, it is:
```
Rectangle [length=5.6, width=7.8, area=0.0, perimeter=0.0]
Rectangle [length=5.6, width=7.8, area=0.0, perimeter=0.0]
Rectangle [length=5.6, width=7.8, area=0.0, perimeter=0.0]
length is: 5.6
width is: 7.8
area is: 43.68
perimeter is: 26.80
```

Need to fix it.

Reply:
done.